atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/utilities/MultiCassandraUtilsV2.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.ete.utilities;
+
+import com.google.common.base.Throwables;
+import com.palantir.docker.compose.connection.Container;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Set;
+import org.awaitility.Awaitility;
+
+public final class MultiCassandraUtilsV2 {
```

Comment:
Exact replica of `MultiCassandraUtils`, but uses `EteExtension` instead of `EteSetup`.